server/server/src/file_service.rs

```diff
@@ -78,18 +78,6 @@ pub async fn change_document_content(
         return Err(None);
     };
 
-    usage_repo::track_content_change(
```

Comment:
it looks to me like deleted files would still be counted towards usage. probably do one of:
* call file_index_repo::change_document_version_and_size here
* update the size to 0 in file_index_repo::delete_file
* exclude deleted files in addition to folders in file_index_repo::get_file_usages